test/solc/CommandLineInterface.cpp

```diff
@@ -25,6 +25,8 @@
 
 #include <test/Common.h>
 #include <test/libsolidity/util/SoltestErrors.h>
+#include <liblangutil/SemVerHandler.h>
+#include <liblangutil/Scanner.h>
```

Comment:
Does not look like we use `Scanner` in this file. Is this actually necessary here?

Reply:
This was marked as resolved but `Scanner.h` is still included here...